src/main/java/org/jboss/pnc/bifrost/endpoint/Rest.java

```diff
@@ -24,36 +24,26 @@
     @GET
     @Path("/text")
     @Produces(MediaType.TEXT_PLAIN)
-    Response getAllLines(
-            @QueryParam("matchFilters") String matchFilters,
-            @QueryParam("prefixFilters") String prefixFilters,
-            @QueryParam("afterLine") Line afterLine,
-            @QueryParam("direction") Direction direction,
-            @QueryParam("maxLines") Integer maxLines,
-            @QueryParam("follow") boolean follow,
-            @QueryParam("timeoutProbeString") String timeoutProbeString); //if string is defined the server is sending given string as a connection probe. The string is printed with the logs.
+    Response getAllLines(@QueryParam("matchFilters") String matchFilters, @QueryParam("prefixFilters") String prefixFilters,
+            @QueryParam("afterLine") Line afterLine, @QueryParam("direction") Direction direction,
+            @QueryParam("maxLines") Integer maxLines, @QueryParam("follow") boolean follow,
+            @QueryParam("timeoutProbeString") String timeoutProbeString); // if string is defined the server is sending given
```

Comment:
It's been more readable before the change.

Reply:
Proposed fix upstream: https://github.com/project-ncl/ide-config/pull/1